crates/storage_impl/Cargo.toml

```diff
@@ -16,6 +16,7 @@ v1 = ["api_models/v1", "diesel_models/v1", "hyperswitch_domain_models/v1"]
 v2 = ["api_models/v2", "diesel_models/v2", "hyperswitch_domain_models/v2"]
 payment_v2 = ["hyperswitch_domain_models/payment_v2", "diesel_models/payment_v2"]
 customer_v2 = ["api_models/customer_v2", "diesel_models/customer_v2", "hyperswitch_domain_models/customer_v2"]
+payment_methods_v2 = ["diesel_models/payment_methods_v2"]
```

Comment:
Please update `scripts/ci-checks-v2.sh` to include this feature, wherever `storage_impl` crate is being checked.